app/plot_app/main.py

```diff
@@ -30,7 +30,7 @@
 
     plots = []
     start_time = timer()
-
+    print("StatisticsPlots!")
```

Comment:
Preferably this should not appear in the history

Reply:
This two print statement I have checked my repo it doesn't appear, does this was the old commit?

Reply:
removed from history.